components/renku_data_services/metrics/staging.py

```diff
@@ -0,0 +1,86 @@
+"""Implementation of staging metrics service."""
```

Comment:
suggestion: I think we can rename this to `core.py`. Just to stick to some of the loose conventions we have and also to keep the file names between different components more uniform.